torchvision/prototype/transforms/functional/_type_conversion.py

```diff
@@ -23,3 +24,23 @@ def decode_video_with_av(encoded_video: torch.Tensor) -> Tuple[torch.Tensor, tor
 
 def label_to_one_hot(label: torch.Tensor, *, num_categories: int) -> torch.Tensor:
     return one_hot(label, num_classes=num_categories)  # type: ignore[no-any-return]
+
+
+def to_image_tensor(image: Union[torch.Tensor, PIL.Image.Image, np.ndarray], copy: bool = False) -> torch.Tensor:
+    if isinstance(image, torch.Tensor):
+        if copy:
+            return image.clone()
+        else:
+            return image
+
+    return _F.to_tensor(image)
+
+
+def to_image_pil(image: Union[torch.Tensor, PIL.Image.Image, np.ndarray], copy: bool = False) -> PIL.Image.Image:
+    if isinstance(image, PIL.Image.Image):
+        if copy:
+            return image.copy()
+        else:
+            return image
+
+    return _F.to_pil_image(to_image_tensor(image, copy=False))
```

Comment:
I am uncertain if it makes sense to do:
```suggestion
    return _F.to_pil_image(to_image_tensor(image, copy=copy))
```

Thoughts?